packages/engine/Source/Renderer/ShaderBuilder.js

```diff
@@ -92,7 +92,7 @@ Object.defineProperties(ShaderBuilder.prototype, {
    * the vertex shader.
    *
    * @memberof ShaderBuilder.prototype
-   * @type {Object.<String, Number>}
```

Comment:
We still have "Number" here. It is an unusual pattern, perhaps your search pattern missed it?